src/osd/osd_types.h

```diff
@@ -4148,7 +4150,27 @@ class ObjectModDesc {
     ENCODE_FINISH(bl);
   }
   void rollback_extents(
-    version_t gen, const std::vector<std::pair<uint64_t, uint64_t> > &extents) {
+   const version_t gen,
+   const std::vector<std::pair<uint64_t, uint64_t>> &extents,
+   const uint64_t object_size,
+   const std::vector<shard_id_set> &shards) {
```

Comment:
nit: bracing. How about

```suggestion
   const std::vector<shard_id_set> &shards)
{
```
?

Reply:
Disagree - while I prefer your style the rest of src/osd/osd_types.h is written with the braces on the same line as the function. I think consistency of style in a file is more important, even if this file has a different style to most of the rest of Ceph!

Reply:
Ouch, sorry for missing that. I agree the spatial consistency is more important.